.monitor.mk

```diff
@@ -0,0 +1,10 @@
+.PHONY: mon
+mon: mon-server mon-client
+
+.PHONY: mon-server
+mon-server:
```

Comment:
Adding it here does not automatically add it to circleci. Please add these binaries to the circleci config.